certificate-service/domain/src/main/java/se/inera/intyg/certificateservice/domain/common/exception/CertificateActionForbidden.java

```diff
@@ -1,8 +1,21 @@
 package se.inera.intyg.certificateservice.domain.common.exception;
 
+import java.util.Collections;
+import java.util.List;
+import lombok.Getter;
+
+@Getter
 public class CertificateActionForbidden extends RuntimeException {
 
+  private final List<String> reason;
+
   public CertificateActionForbidden(String message) {
```

Comment:
Suggest we remove the default-constructor and force all places to provide reason.